train.py

```diff
@@ -456,7 +464,11 @@ def main(args):
 		if args.fp16:
 			model = models.InputOutputTypeCast(model.to(torch.float16), dtype = torch.float16)
 
-		waveform_input = torch.rand(args.onnx_sample_batch_size, args.onnx_sample_time, device = args.device)
+		if args.onnx_waveform_input:
+			waveform_input = torch.load(args.onnx_waveform_input).to(device = args.device).squeeze(1)
```

Comment:
why is `squeeze` needed?

Reply:
The waveform was exported with shape tree dimensions. Should I check the frontend in model params and assert this case?

Reply:
please add a comment in the code about this

Reply:
done